packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -184,7 +184,7 @@ contract PerpetualPositionManager is FundingRateApplier {
         fees()
         nonReentrant()
     {
-        require(collateralAmount.isGreaterThan(0), "Invalid collateral amount");
+        require(collateralAmount.isGreaterThan(0));
```

Comment:
Are these the same require messages that were removed in #2278? Will that mean that once these get merged on top of one another, we'll still be over the size limit?

Reply:
Yes so I merged #2278 and now this PR is 150 bytes over.